### PR TITLE
improve(Dataworker): Make logs more clear when choosing to skip exchange rate updates

### PR DESCRIPTION
**Pull request by nicholaspai**
- Add details to logs
- Fix one bug to skip duplicate tokens in `_updateOldExchangeRates`.

### CODE REVIEW

File: src/dataworker/Dataworker.ts [hunk 1]
```diff
@@ -1554,7 +1554,7 @@ export class Dataworker {
     const l1TokensWithPotentiallyOlderUpdate = expectedTrees.poolRebalanceTree.leaves.reduce((l1TokenSet, leaf) => {
       const currLeafL1Tokens = leaf.l1Tokens;
       currLeafL1Tokens.forEach((l1Token) => {
-        if (!l1TokenSet[l1Token] && !updatedL1Tokens.has(l1Token)) {
+        if (!l1TokenSet.includes[l1Token] && !updatedL1Tokens.has(l1Token)) {
```
Comment (james-a-morris) [inline src/dataworker/Dataworker.ts:1557]: OOC `l1TokenSet` is an array - was this previously working or is this a bug fix?

**Review by james-a-morris — APPROVED** [review]
Looks good. Nice catch on the variable spelling. One question about the change from [l1Token] to .includes(l1Token)

File: src/dataworker/Dataworker.ts (hunk 1 above)
Comment (mrice32) [inline src/dataworker/Dataworker.ts:1557]:
```suggestion
        if (!l1TokenSet.includes(l1Token) && !updatedL1Tokens.has(l1Token)) {
```

**Review by mrice32 — APPROVED** [review]
Only comment is a syntax bug. Otherwise, LGTM!

File: src/dataworker/Dataworker.ts (hunk 1 above)
Comment (nicholaspai) [reply src/dataworker/Dataworker.ts:1557]: this is a bug fix. it should work now no?

**Pull request merged**